lib/customer-deposit-wallet/src/Cardano/Wallet/Deposit/IO.hs

```diff
@@ -228,7 +234,7 @@ withWalletDBVar
                         [ walletTip
                         , Read.GenesisPoint
                         ]
-                , rollForward = rollForward w
+                , rollForward = rollForward w wtc
                 , rollBackward = rollBackward w
```

Comment:
For the sake of showing sync progress this is probably what we want (or practically what we want), but techincally the `rollBackward` would also update the wallet tip.